### PR TITLE
Make a Dockerfile

### PR DESCRIPTION
**Pull request by theblobscp**
Docker is cool

### CODE REVIEW

**Comment by DotWith** [comment]
How does this work? I've never used Dockerfile.

**Comment by theblobscp** [comment]
Fire ship has a good video on it, watch it and you’ll probably understand it 
Link: https://www.youtube.com/watch?v=Gjnup-PuquQ

**Comment by DotWith** [comment]
> Fire ship has a good video on it, watch it and you’ll probably understand it Link: https://www.youtube.com/watch?v=Gjnup-PuquQ

This needs to be merged

**Comment by theblobscp** [comment]
I'm still working on some bugs, but thanks for the feedback

**Comment by AbnormalPoof** [comment]
why only linux

**Comment by DotWith** [comment]
> why only linux (in the build command)

Docker is a Linux **based** thing

**Comment by AbnormalPoof** [comment]
oops
what about html5

**Review by AbnormalPoof — CHANGES_REQUESTED** [review]
maybe add this after linux idk

```
# Generate the built game for html5
CMD [ "haxelib", "run", "lime", "test", "html5" ]
```

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 export/
 .vscode/
-APIStuff.hx
```
Comment (Steviegt6) [inline .gitignore:3]: If you're gonna do this, you should instead use environment variables, not a class containing two strings.

**Review by Steviegt6 — CHANGES_REQUESTED** [review]
Using a docker for this is actually really dumb. Use a GitHub workflow for building. You can use encrypted environment variables to avoid leaking API keys (though they're easy to retrieve regardless due to source compilation & I believe Ninja is taking a different approach in the private fork).

**Comment by theblobscp** [comment]
> 
> 
> maybe add this after linux idk
> 
> ```
> # Generate the built game for html5
> CMD [ "haxelib", "run", "lime", "test", "html5" ]
> ```

CMD can only be used once

File: .gitignore (hunk 1 above)
Comment (theblobscp) [reply .gitignore:3]: Do you have a block of code I can add to it

**Comment by theblobscp** [comment]
Also html would be useless, as it auto-runs it. Docker is for CLI, not GUI.

**Comment by theblobscp** [comment]
I realized that for some reason the builds don’t work due to an error in the final build. I’m gonna close the pr until I can find a solution to this. If you need to test it, git pod is where I did it.

**Pull request closed without merging**